bucket/kotatogram-beta.json

```
@@ -0,0 +1,41 @@
+{
+    "version": "1.4.9",
+    "description": "Experimental Telegram Desktop fork. Kotatogram Desktop, being based on Telegram Desktop, has all its features, but it also has some more useful and cosmetic features(Beta Edition).",
+    "homepage": "https://kotatogram.github.io",
+    "license": {
+        "identifier": "GPL-3.0-with-OpenSSL-exception",
+        "url": "https://github.com/kotatogram/kotatogram-desktop/blob/dev/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kotatogram/kotatogram-desktop/releases/download/k1.4.9/1.4.9-win64-portable.zip",
+            "hash": "4d50fbca6b1129c4b52e3933b7931ac878c271f77e0472f8d0a40c6ff76d20fa"
+        },
+        "32bit": {
+            "url": "https://github.com/kotatogram/kotatogram-desktop/releases/download/k1.4.9/1.4.9-win32-portable.zip",
+            "hash": "45ffdf047e5d1847242b4059dde8d439288bb9af08119b2eac7363f807b3d44a"
+        }
+    },
+    "bin": "Kotatogram.exe",
+    "shortcuts": [
+        [
+            "Kotatogram.exe",
+            "Kotatogram"
+        ]
+    ],
+    "persist": "tdata",
+    "checkver": {
+        "url": "https://github.com/kotatogram/kotatogram-desktop/releases",
+        "regex": "Version ([\\d.]+) beta"
```

```suggestion
        "regex": "Version ([\\d.]+)( beta)?"
```
Include **stable** versions as well. This is the convention for official buckets,
because for users who are using beta channels, usually what they want is the latest features, no matter iit is thoroughly tested or not.